transforms/mods_to_dc_oai.xsl

```diff
@@ -242,59 +250,80 @@
 	<xsl:template match="mods:temporal[@point != 'start' and @point != 'end']">
 		<xsl:value-of select="."/>
 	</xsl:template>
+	
 	<xsl:template match="mods:genre">
 		<xsl:choose>
 			<xsl:when test="@authority = 'dct'">
 				<dc:type>
 					<xsl:value-of select="."/>
 				</dc:type>
 			</xsl:when>
+			<xsl:when test="../mods:typeOfResource">
+					<xsl:apply-templates select="mods:typeOfResource"/>
+			</xsl:when>	
 			<xsl:otherwise>
 				<dc:type>
-					<xsl:value-of select="."/>
+					Image
 				</dc:type>
-				<xsl:apply-templates select="mods:typeOfResource"/>
 			</xsl:otherwise>
 		</xsl:choose>
 	</xsl:template>
+	
+	<xsl:template match="mods:mods[not(mods:genre)]">
```

Comment:
@markpbaggett this template can be dropped - the logic here duplicates the logic that you set up in the main for-each loop on lines 76.